pyproject.toml

```diff
@@ -40,6 +40,8 @@ grpcio-status = ">=1.56.0"
 faiss-cpu = "^1.7.4"
 sentence-transformers = "^2.2.2"
 flagembedding = "^1.1.1"
+babel = "^2.12.1"
+torch = ">=2.0.0, !=2.0.1"
```

Comment:
why put more dependencies here? we should revisit all the dependency for SimilarValueTool and make our SDK more lightweight.